### PR TITLE
Style the bookstore

### PR DESCRIPTION
**Pull request by Estherstarr**
In this milestone, I implemented the following

- [ ] Style the bookstore using the mock displayed in the Zeplin

### CODE REVIEW

File: src/components/Navbar.js [hunk 1]
```diff
@@ -3,12 +3,23 @@ import { Link } from 'react-router-dom';
...
-      <Link to="/">Books</Link>
-      <Link to="/categories">
-        Categories
-      </Link>
-    </nav>
+    <div className="navigation">
+      <div className="topnav__bar">
+        <div className="navlinks">
+          <Link to="/" className="logo__text">
+            Bookstore CMS
```
Comment (RolandM99) [inline src/components/Navbar.js]: - [ ] As requested by the project requirement, your style should fit exactly the original design provided by Zepplin. Kindly, make sure that your navbar looks as the navbar from the original design. Kindly, add the user icon as on the original design and also fix the gap between BOOKS and CATEGORIES menu items.

| Your Style | Original Style |
| ----------- | --------------- |
|image|image|

**Review by RolandM99 — CHANGES_REQUESTED** [review]
Hi @Estherstarr , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


### To highlight 💯 
- Good commit message ✔️ 
- No Linter errors ✔️ 
- Follow Github flow ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on @Rolandm99 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Estherstarr** [comment]
@RolandM99 Thank you for the review
Changes made
- [ ] Add the user icon as on the original design and also fix the gap between BOOKS and CATEGORIES menu items

**Review by BenMukebo — APPROVED** [review]
### Hi @Estherstarr, 👋 

# STATUS: Approved  🎉 

Great work on making the changes requested by a previous reviewer :1st_place_medal:  
 Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 




## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [OPTIONAL] Please, add a meaningful  `Project description`. Remember A project description is a high-level overview of why you're doing a project. The document explains a project's objectives and essential qualities. Is an essential document that provides your audience with a comprehensive overview of the project. Think of it as the elevator pitch that focuses on what and why without delving into how.

- The README file It is the first file a person will see when they encounter your project, so it should be fairly brief but detailed. It will make your project stand out from a bunch of others.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**